co2eq/purchase/consumerpriceindices.yml

```diff
@@ -0,0 +1,520 @@
+AU:
```

Comment:
could you simplify the file to only push the numbers required (to lower the size of the file)?

Reply:
Right now the file has for each country only the yearly averages so I don't understand what you mean. 
In the notion the v0 says: "Use only annual country CPI (COICOP agnostic)", doesn't that mean yearly averages for every country?

Reply:
got it. Let's keep it like that then.
Are you able to separate the AVERAGE key from the countries by having this structure:
- countries
  AT:
    2011: ... # <-- note the key is not a string but a number
- average:
    2011: ...

Reply:
I think this is the last outstanding issue and then we should be able to merge!

Reply:
i think this is the last issue remaining and then we should be able to merge!